ddtrace/internal/writer.py

```diff
@@ -79,15 +76,13 @@ def __init__(
         super(AgentWriter, self).__init__(
             interval=processing_interval, exit_timeout=shutdown_timeout, name=self.__class__.__name__
         )
+        self.url = url
```

Comment:
```suggestion
        self.agent_url = url
```
for compat?

Reply:
:+1: `agent_url` is a better variable name in general 😆